metpy/plots/declarative.py

```diff
@@ -657,29 +660,21 @@ def draw(self):
                 self.ax.set_extent(area, ccrs.PlateCarree())
 
             # Use the set title or generate one.
-            title = self.title or ', '.join(plot.name for plot in self.plots)
+            title = self.title or ', \n'.join(plot.name for plot in self.plots)
```

Comment:
So this would take all the plot names and join them together with a comma followed by a space, followed by a newline. Is that really what's intended?

Reply:
This was the best I could come up with to get a "clean" title. Example below...
![test](https://user-images.githubusercontent.com/20846805/59377563-14404a80-8d18-11e9-8fb0-53886535e4b6.png)

Reply:
Fair enough. Maybe just remove the space since it's redundant with the '\n'.